main.nf

```diff
@@ -1533,11 +1669,12 @@ if (!params.skipAlignment) {
   dupradar_results = Channel.from(false)
   preseq_results = Channel.from(false)
   featureCounts_biotype = Channel.from(false)
+  rsem_logs = Channel.from(false)
 }
 
 
 /*
- * STEP 11 - Transcriptome quantification with Salmon
+ * STEP 15 - Transcriptome quantification with Salmon
```

Comment:
This is a good idea to divide the codebase in steps, however, these tend to add maintainence as well and eventuallly become out of sync. Maybe we could start relying on `NOTE` or simply `STEP` for this purpose?